src/components/SubRequirement.vue

```diff
@@ -88,9 +88,12 @@
             :subReqCourseId="id"
             :crseInfoObjects="subReqCrseInfoObjects"
             :subReqCourseObjectsNotTakenArray="subReqCourseObjectsNotTakenArray"
+            :subReqCoursesNotTakenArray="subReqCoursesNotTakenArray"
```

Comment:
why are we passing both around?

Reply:
I'll rename the variables. But the subReqCourseObjectsNotTakenArray is fetched from the DB and only return the first four. The functions to fetch those courses could be moved to `src/components/IncompleteSubReqCourse.vue, but I think it might be used in SubRequirement later